app/controllers/FileChecksController.scala

```diff
@@ -78,12 +115,19 @@ class FileChecksController @Inject() (
             reference,
             applicationConfig.frontEndInfo,
             request.token.name,
-            isJudgmentUser = true
+            isJudgmentUser
           )
         ).uncache()
       } else {
-        Ok(views.html.judgment.judgmentFileChecksProgress(consignmentId, reference, applicationConfig.frontEndInfo, request.token.name)).uncache()
+        if (isJudgmentUser) {
+          Ok(views.html.judgment.judgmentFileChecksProgress(consignmentId, reference, applicationConfig.frontEndInfo, request.token.name)).uncache()
+        } else {
+          Ok(views.html.standard.fileChecksProgress(consignmentId, reference, applicationConfig.frontEndInfo, request.token.name))
+            .uncache()
+        }
       }
+    }).recover { case exception: Exception =>
```

Comment:
Does the user still see an error on the page if this is the case?

Reply:
Yes, any exception that is thrown will be caught and user is taken to the error page.